dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Deploy TiDB in Kubernetes Using kind
+summary: Learn how to deploy a TiDB cluster in Kubernetes using kind.
+category: how-to
+---
+
+# Deploy TiDB in Kubernetes Using kind
+
+This tutorial shows how to deploy [TiDB Operator](https://github.com/pingcap/tidb-operator) and a TiDB cluster in Kubernetes on your laptop (Linux or macOS) using [kind](https://kind.sigs.k8s.io/).
+
+kind is a tool for running local Kubernetes clusters using Docker containers as cluster nodes. It is developed for testing local Kubernetes clusters, initially targeting the conformance tests. The Kubernetes cluster version depends on the node image that your kind uses, and you can specify the image to be used for the nodes and choose any other published version. Refer to [Docker hub](https://hub.docker.com/r/kindest/node/tags) to see available tags.
```

Comment:
```suggestion
kind is a tool for running local Kubernetes clusters using Docker containers as cluster nodes. It is developed for testing local Kubernetes clusters, initially targeting the conformance tests. The Kubernetes cluster version depends on the node image that your kind uses, and you can specify the image to be used for the nodes and choose any published version. Refer to [Docker hub](https://hub.docker.com/r/kindest/node/tags) to see available tags.
```
@yikeke  If we use "and" to join the sentence, `other` in `any other` should be removed here.
FYI, https://stackoverflow.com/questions/56046451/use-kind-to-install-different-kubernetes-version